topics/materials-science/tutorials/muspinsim/workflows/6_Dissipation.ga

```diff
@@ -0,0 +1,204 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Example to show effect of dissipation constant. Compare effect against theory",
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "Muon Spectroscopy Computational Project",
+            "url": "https://muon-spectroscopy-computational-project.github.io"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "name": "Muspinsim Example 6 - Dissipation (imported from uploaded file)",
```

Comment:
This workflow fails when plotting - it looks like it may be trying to read the log file instead of the .dat file outputted from the previous step.
![image](https://user-images.githubusercontent.com/90245114/205893964-2dc26f42-87a2-4378-9261-f61e531f35d5.png)

Reply:
@patrick-austin No I didn't notice that part, just tried again but it seems they still wont run as at least one of the fields are always blank after clearing the history and don't have anything to select to fill them. Not sure if its my misunderstanding on how to use them though.

Reply:
Ah sorry I might not have been clear, the history shouldn't be **entirely** empty, it should still contain `dissipation_theory.dat` for 6 and `experiment.dat` for 7 as you will need these files. But any other text files in your history will interfere with the auto-selection and you'll need to select the right file manually.

Reply:
Thanks, it works now. The warnings are good idea.